src/utils/rgutils.py

```diff
@@ -204,3 +205,83 @@ def depth_exists(id, connector):
         id[0]) + " and dataformat = 'depth');"
     exists = connector.execute(check_dataformat, fetch_all=True)
     return exists[0][0]
+
+
+def process_posenet_result(
+        pose_prediction,
+        model_id,
+        artifact_id,
+        db_connector):
+    table = "artifact_result"
+    PART_NAMES = [
```

Comment:
Put as global variable above the function, or even better into a separate file, e.g. called `constants.py`